layout_opentitan.ld

```diff
@@ -1,9 +1,13 @@
 /* Layout for the RISC-V 32 boards, used by the examples in this repository. */
 
 MEMORY {
-  /* The TBF header region is 32 bytes (0x20) */
-  FLASH (rx) : ORIGIN = 0x20030020, LENGTH = 32M
-  SRAM (rwx) : ORIGIN = 0x10000000, LENGTH = 512K
+  /*
+   * The TBF header can change in size so use 0x40 combined with
+   * --protected-region-size with elf2tab to cover a header upto that
+   * size.
+   */
+  FLASH (rx) : ORIGIN = 0x20030040, LENGTH = 32M
+  SRAM (rwx) : ORIGIN = 0x10002800, LENGTH = 512K
```

Comment:
You don't mention why you are changing this addresses in the PR, can you explain why this is being changed.